modules/node-shared/src/main/scala/org/tessellation/node/shared/infrastructure/snapshot/CurrencySnapshotAcceptanceManager.scala

```diff
@@ -47,6 +48,7 @@ object CurrencySnapshotAcceptanceManager {
       lastSnapshotContext: CurrencySnapshotContext,
       lastActiveTips: SortedSet[ActiveTip],
       lastDeprecatedTips: SortedSet[DeprecatedTip],
+      ordinal: SnapshotOrdinal,
```

Comment:
I see that on `develop` branch we already have a `snapshotOrdinal` property here. Would you mind doing the same here (name/order)? I think it would be easier for later merging with develop